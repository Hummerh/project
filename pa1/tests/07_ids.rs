abc ABC abc123 abc_123 123 abc_
a_b_c_d_e_f_g_h_i_j_k_l_m_n_o_p_q_r_s
