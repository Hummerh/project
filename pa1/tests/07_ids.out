ID(abc)
ID(ABC)
ID(abc123)
ID(abc_123)
LITDEC(123)
ID(abc_)
ID(a_b_c_d_e_f_g_h_i_j_k_l_m_n_o_p_q_r_s)
